tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml

```diff
@@ -189,7 +189,7 @@ jobs:
       displayName: 'Run build script'
       inputs:
         filename: 'build.bat'
-        arguments: ' --use_cuda --cuda_home="C:\local\cuda_10.0.130_win10"
+        arguments: ' --use_cuda --cuda_home="C:\local\cuda_10.0.130_win10_copy"
```

Comment:
why _copy ? what's different vs before?

Reply:
new TRT DLLs will be copied to CUDA directory. That's why a copy of cuda is created on the CI machines before the PR is merged, otherwise other people's PR will fail. Will remove the copy version when the PR is merging.

Reply:
Could have named it more descriptively and include trt version in the name. Then you don't need to make another PR to change it again.
Would prefer if we don't checkin yml files that reference a dir name with _copy in it.